staging_docs/admin/guides/01-rbac.md

```diff
@@ -0,0 +1,379 @@
+# Role Based Access Control
+
+!!! warning
+    This feature is a Tech Preview and may be subject to change.
+
+
+!!! note
+    User `admin` automatically gets all RBAC permissions.
+
+
+RPM plugin presents Role Based Access for these endpoints :
+
+- RPM Alternate Content Source (`/pulp/api/v3/acs/rpm/rpm/`)
+- RPM Distributions (`/pulp/api/v3/distributions/rpm/rpm/`)
+- RPM Publications (`/pulp/api/v3/publications/rpm/rpm/`)
+- RPM Remotes (`/pulp/api/v3/remotes/rpm/rpm/` and `/pulp/api/v3/remotes/rpm/uln`)
+- RPM Repository (`/pulp/api/v3/repositories/rpm/rpm/`)
+- RPM Copy (`/pulp/api/v3/rpm/copy/`)
+
+!!! note
+    Content is secured too, but the only condition right now is the authenticated user.
+
+
+## Default Roles
+
+RPM plugin ships a few default roles described bellow.
+
+### Viewer Role
+
+This role contains only `view` permission for an endpoint. Using this permission you can control who can
+view specific content for example a repository.
+
+=== "Check a Role"
+
+    ```shell
+     pulp role show --name "rpm.rpmremote_viewer"
+    ```
+
+=== "Output"
```

Comment:
IIRC, this stuff isn't expected to render in Github, but it works w/ our docs tooling, right?

Reply:
Correct - renders correctly under `pulp-docs serve` and when delivered by the docs-site.